ui/src/variables/utils/resolveSelectedValue.ts

```diff
@@ -1,5 +1,3 @@
-import {VariableMapObject} from 'src/types'
```

Comment:
Let's remove this this type from the `variables` file (and also any other reference to it)

Reply:
it still gets used, just not externally

Reply:
Do we still want to use it? Does it still serve a purpose? I added that type in to enforce variable mapping types.

Reply:
i found it useful. i was even going to extend it to more parts of the app, but typescript decided to... die

Reply:
🤷‍♂